qubes/tests/integ/dispvm.py

```diff
@@ -221,6 +225,118 @@ def test_010_simple_dvm_run(self):
         finally:
             self.loop.run_until_complete(dispvm.cleanup())
 
+    def test_011_dvm_run_preload_invalid_max(self):
+        # TODO: couldn't make the assert raise on unit tests.
+        cases_invalid = ["a", "-1", "1 1"]
+        for value in cases_invalid:
+            with self.subTest(value=value):
+                with self.assertRaises(qubes.exc.QubesValueError):
+                    self.appvm.features["preload-dispvm-max"] = value
```

Comment:
Did you mean `self.disp_base` instead of `self.appvm`? Same applies to few more cases below.